### ISSUE TITLE
Nuget.Server 3.3.0 shows a "Duplicate 'Assembly###' attribute" error upon starting 

### ISSUE THREAD

**Issue by mtrellet** [NuGet/NuGetGallery#7396]
### Describe the bug
When following the tutorial on https://docs.microsoft.com/en-us/nuget/hosting-packages/nuget-server and installing Nuget.Server 3.3.0, I get the following errors when I try to `Start the project` (see also the Screenshot section).

```Severity	Code	Description	Project	File	Line	Suppression State
Error	CS0579	Duplicate 'ComVisible' attribute	XXXNugetServer	C:\Users\Mikael Trellet\###\###\###NugetServer\Properties\AssemblyInfo.cs	20	Active
Error	CS0579	Duplicate 'AssemblyTrademark' attribute	XXXNugetServer	C:\Users\Mikael Trellet\###\###\###NugetServer\Properties\AssemblyInfo.cs	14	Active
...
```

### To Reproduce
Follow the steps on https://docs.microsoft.com/en-us/nuget/hosting-packages/nuget-server and select the latest stable release of `Nuget.Server` (3.3.0).

### Expected behavior
A web page should open on localhost displaying a success message like for a previous version (see Screenshot section).

### Screenshots
![image](https://user-images.githubusercontent.com/715654/62298767-ed7ad680-b473-11e9-8649-76f64e6de99b.png)
![image](https://user-images.githubusercontent.com/715654/62298862-22872900-b474-11e9-8eb2-fce00b143035.png)

### Additional context
Using Visual Studio Professional 2019 (v16.1.6)

**Comment by joelverhagen** [comment]
Hello, I attempted the steps you mentioned and did not see the errors you mentioned. I tried both 3.3.0 which is what you mentioned and 3.1.2 which is in your screenshot.

Could you ZIP up the project you created and upload it to this issue? For reference, here are the two projects I made:

NuGet.Server 3.3.0 - [WebApplication2.zip](https://github.com/NuGet/NuGetGallery/files/3478391/WebApplication2.zip)
NuGet.Server 3.1.2 - [WebApplication3.zip](https://github.com/NuGet/NuGetGallery/files/3478392/WebApplication3.zip)

Perhaps you can look at these two projects and understand where the difference lies?

**Comment by joelverhagen** [comment]
Please feel free to re-open this issue if you are still running into the problem. As mentioned above I was unable to reproduce the issue.

**Comment by mtrellet** [comment]
Hello, thanks a lot for your answer and provided projects. Sorry for the late answer, I was away from my work computer for a couple of weeks.

> I tried both 3.3.0 which is what you mentioned and 3.1.2 which is in your screenshot.

The last screenshot might have been misleading, it works properly for version 3.1.2 but I still get the error for version 3.3.0.

> NuGet.Server 3.3.0 - WebApplication2.zip

Extracting the project and running it leads to this page in Chrome:

![image](https://user-images.githubusercontent.com/715654/63256008-027aa680-c277-11e9-9f92-2a76d5eaa882.png)

> Perhaps you can look at these two projects and understand where the difference lies?

I've made a quick diff between your project files and mine, apart from the framework version targeted (you:`4.6` vs mine:`4.6.1`) I could not spot anything obvious. 
However, I've seen that your `Packages` directory is empty, mine contains the different nuget packages that are installed for the project. 
To test, I've made a new project following the usual steps and just deleted the content of the `Packages` directory and it worked..! It was stupid but the conflicts certainly comes from there.  I'd assume a conflict with the Nuget packages installed in my global directory and the local ones..? 

In any case, I also tried with the latest version of Nuget.Server (v3.4.1) and it worked without issue, no package were add to the `packages` directory and I could get a proper landing page upon running.

I'm a bit puzzled by why this directory gets populated for a version and not for another but at least the issue is solved..!

Thanks a lot for your time.
